servicemesh-install/topics/install-remove.adoc

```diff
@@ -1,44 +1,81 @@
 [[removing-service-mesh]]
 = Removing {ProductName}
 
-The following process removes the {ProductShortName} from an existing installation. Log in as a cluster admin and run these commands from any host with access to the cluster.
+The following process allows you to remove the {ProductShortName} from an existing {product-title} instance. Log in as a cluster admin and run these commands from any host with access to the cluster.
```

Comment:
Not sure why you changed "removes" to "allows you to remove".  IBM Style says to use "allow" with caution and I don't think there's anything about this action that is granting abilities. You do these steps, things are removed.

Reply:
I changed this to read:

"Follow this process to remove the {ProductShortName} from an existing {product-title} instance."

The process has no agency. The user follows the process.